docs/articles_en/openvino-workflow/model-optimization.rst

```diff
@@ -13,57 +10,103 @@ Model Optimization Guide
-  , a suite of advanced methods for training-time model optimization within the DL
-  framework, such as PyTorch and TensorFlow 2.x. It supports methods like
-  Quantization-aware Training, Structured and Unstructured Pruning, etc.
+.. image:: ../assets/images/WHAT_TO_USE.svg
 
-- :doc:`Weight Compression <model-optimization-guide/weight-compression>`,
-  an easy-to-use method for Large Language Models footprint reduction and inference acceleration.
 
-.. note::
+| :doc:`Post-training Quantization <model-optimization-guide/quantizing-models-post-training>`:
+|      designed to optimize deep learning models by applying 8-bit integer quantization. Being
+       the easiest way to optimize a model it does not require its retraining or fine-tuning
+       but may result in a drop in accuracy. If the accuracy-performance tradeoff is not
+       acceptable, Training-time Optimization may be a better option.
+
+| :doc:`Training-time Optimization <model-optimization-guide/compressing-models-during-training>`:
+|      involves a suite of advanced methods such as Structured or Unstructured Pruning, as well
+       as Quantization-aware Training. This kind of optimization requires the use of the model's
+       original framework, for NNCF, it is either PyTorch or TensorFlow.
+
+| :doc:`Weight Compression <model-optimization-guide/weight-compression>`:
+|      an easy-to-use method for Large Language Model footprint reduction and inference
+       acceleration.
+
 
-   OpenVINO also supports optimized models (for example, quantized) from source
-   frameworks such as PyTorch, TensorFlow, and ONNX (in Q/DQ; Quantize/DeQuantize format).
-   No special steps are required in this case and optimized models can be converted to
-   the OpenVINO Intermediate Representation format (IR) right away.
+To learn about the full scope of the framework, its installation, and technical details, visit
+both `the NNCF repository <https://github.com/openvinotoolkit/nncf?tab=readme-ov-file>`__ and
+`NNCF API documentation <https://openvinotoolkit.github.io/nncf/autoapi/nncf/>`__.
 
-Post-training Quantization is the fastest way to optimize an arbitrary DL model and should
-be applied first, but it is limited in terms of achievable accuracy-performance trade-off.
-The recommended approach to obtain OpenVINO quantized model is to convert
-`a model from original framework <https://huggingface.co/models>`__ to ``ov.Model``
-and ensure that the model works correctly in OpenVINO, for example, by calculating the
-model metrics. Then, ``ov.Model`` can be used as input for the ``nncf.quantize()``
-method to get the quantized model or as input for the ``nncf.compress_weights()``
-method to compress weights of Large Language Models (see the diagram below).
 
-In case of unsatisfactory accuracy or performance after Post-training Quantization,
-Training-time Optimization can be used as an option.
 
 .. image:: ../assets/images/DEVELOPMENT_FLOW_V3_crunch.svg
```

Comment:
We need to extend it by the Weights Compression

Reply:
There is weight compression in the top row, do you think we should separate it to be more visible, or maybe break it down to present more details?

Reply:
Disregard my comment. I think I missed it or smth's confused me.